server/src/shared/genanki/package.ts

```diff
@@ -54,7 +54,7 @@ export class Package {
 
             zip.file("media", JSON.stringify(media_info));
 
-            zip.generateAsync({ type: "blob", mimeType: "application/apkg" }).then(function (content) {
+            zip.generateAsync({ type: "nodebuffer", mimeType: "application/apkg" }).then(function (content) {
```

Comment:
I was getting an error saying something like "blob is not supported in this platform", but changing it the type to `nodebuffer` seems to work